doc/api/crypto.md

```diff
@@ -1468,6 +1471,9 @@ to create the `Decipher` object.
 <!-- YAML
 added: v0.1.94
 changes:
+  - version: v12.0.0
```

Comment:
```suggestion
  - version: REPLACEME
```

Reply:
v10.14.0 right ? Sorry, I just used the `./out/Release/node -v`

Reply:
We do not know yet in which actual release this will land, so instead we write `REPLACEME` and the value is changed when the release that includes this change is done.

Reply:
ah, I misunderstood the `REPLACEMENT`

Reply:
Thank you for being so patient.

Reply:
I think you are the one being patient! Thanks.